### ISSUE TITLE
Wake up on INT pin interrupt from IMU

### ISSUE THREAD

**Issue by zrecore** [femtoduino/ArduinoCore-atsamd21e18a#23]
I can't get the IMU to trigger an interrupt on the INT pin to wake the SAM R21 chip

**Comment by jwestmoreland** [comment]
Alex,

Is this still an issue?

Thanks,
John

**Comment by zrecore** [comment]
@jwestmoreland yes, it's still an issue. I'm not sure if it's due to my lack of understanding on how to get it working.

**Comment by jwestmoreland** [comment]
@zrecore ,

But, it works on the D21?

Regards,
John

**Comment by zrecore** [comment]
@jwestmoreland I'm not sure. I don't fully understand the pin setup, and haven't tried in a long while.

**Comment by jwestmoreland** [comment]
@zrecore ,

OK - thanks for the quick replies.

Best Regards,
John
